vnext/IntegrationTests/DummyTest.js

```diff
@@ -1,37 +0,0 @@
-/**
- * Copyright (c) Microsoft Corporation. All rights reserved.
- * Licensed under the MIT License.
- * @format
- */
-
-'use strict';
-
-const React = require('react');
```

Comment:
>const [](start = 0, length = 5)

Files in this directory are part of the output directory but were accidentally checked in previously, even though they're now under .gitignore.